src/main.zig

```diff
@@ -134,9 +141,10 @@ const Command = struct {
             .state_machine_options = .{
                 // TODO Tune lsm_forest_node_count better.
                 .lsm_forest_node_count = 4096,
-                .cache_entries_accounts = config.cache_accounts_max,
-                .cache_entries_transfers = config.cache_transfers_max,
-                .cache_entries_posted = config.cache_transfers_pending_max,
+                // TODO Bunch these up into a struct?
```

Comment:
I think the current struct is fine; types tend to accumulate per-instance configuration in a flat `Options` struct. (Well, mostly flag, e.g. `ReplicaOptions` has `ReplicaOptions.state_machine_options: StateMachineOptions`).

Reply:
This somewhat contradics "I would put these in a struct so their order can't be mixed up." above :-) 

I guess what I'd do is to define

```
const CachArgs = struct { ... }
```

in ~~`cli.rs`~~ `cli.zig`, use that as an argument to this function, but would _not_ use it as a field of `StateMachineOptions`. SG?

Reply:
Maybe `start()` could just take `state_machine_options` directly?

Above I didn't think we needed another layer to the struct because they were already in a struct, so their order can't be mixed up in the state machine. But right now `start()` just takes all 3 separately, so they can be mixed up there.